src/components/form/templates/array-field-collapsible-item.ts

```diff
@@ -56,14 +59,20 @@ export class CollapsibleItemTemplate extends React.Component {
     }
 
     public render() {
-        const { data, schema, index, formSchema } = this.props;
+        const { data, schema, formSchema } = this.props;
+        const inputs = {
+            header: findTitle(data, schema, formSchema) || 'New item',
+            className: 'limel-form-array-item--object',
+            ref: 'section',
+        };
+
+        if (this.isOpen) {
+            inputs['is-open'] = true;
```

Comment:
We can add this to the object above to get rid of this `if`-statement
```
{
    'is-open': this.isOpen
}
```

Reply:
I couldn't get that to work before but will try again,  maybe I changed something on the way

Reply:
Ah it's the same now, but still doesn't work. Will try something else.